windowing_lib_test.py

```diff
@@ -0,0 +1,65 @@
+import windowing as wl
+import numpy as np
+import pytest
+
+windowed_data = [[[[1, 1, 1, 1, 1], [1, 1, 1, 1, 1, ], [1, 1, 1, 1, 1], [1, 2, 3, 4, 5]], [
```

Comment:
Could you make this local to a test since it seems as though it's only used in 1 test? Is there a benefit to making it global?